grid-client/deployer/tf_plugin_client.go

```diff
@@ -199,16 +200,25 @@ func NewTFPluginClient(
 		tfPluginClient.relayURL = relayURL
 	}
 
-	// default rmbTimeout is 10
+	// default rmbTimeout is 60
 	if rmbTimeout == 0 {
-		rmbTimeout = 10
+		rmbTimeout = 60
 	}
 	tfPluginClient.RMBTimeout = time.Second * time.Duration(rmbTimeout)
 
 	ctx, cancel := context.WithCancel(context.Background())
 	tfPluginClient.cancelRelayContext = cancel
 
-	rmbClient, err := peer.NewRpcClient(ctx, tfPluginClient.mnemonicOrSeed, manager, peer.WithRelay(tfPluginClient.relayURL), peer.WithSession(sessionID), peer.WithKeyType(keyType))
+	var peerOpts []peer.PeerOpt
+	if !rmbInMemCache {
+		peerOpts = append(peerOpts, peer.WithTwinCache(10*60*60)) // in seconds that's 10 hours
+	}
+
+	peerOpts = append(peerOpts, peer.WithRelay(tfPluginClient.relayURL))
+	peerOpts = append(peerOpts, peer.WithSession(sessionID))
+	peerOpts = append(peerOpts, peer.WithKeyType(keyType))
+
```

Comment:
since you already have a fixed set of options you can easily do this instead of the above.
```suggestion
	 peerOpts := []peer.PeerOpt{
	          peer.WithRelay(tfPluginClient.relayURL),
	          peer.WithSession(sessionID)),
	          peer.WithKeyType(keyType),
	 }
	 
	if !rmbInMemCache {
		peerOpts = append(peerOpts, peer.WithTwinCache(10*60*60)) // in seconds that's 10 hours
	}
```

Also it's faster since the compiler can optimise this by already creating the array with the correct capacity